public/js/discover/searchList.js

```diff
@@ -28,40 +28,48 @@ const searchCardTemplate = `
       {{/each}}
     `;
 
-/* global createMarkers, Handlebars, removeMarkers*/
+/* global Handlebars */
 
 /**
- * When the page loads, fetches initial organization data on page load
- * in cards on the list.
+ * Renders the organization data into cards.
+ * @param {JSON} organizations The JSON of organization data to add to the page.
  */
-window.onload = function () {
-  updateResults();
-};
+export function createOrganizationCards(organizations) {
+  // Generate the cards.
+  const renderCards = Handlebars.compile(searchCardTemplate);
+  document.getElementById('search-list').innerHTML = renderCards({organizations: organizations});
 
-/**
- * Requeries for organization data and refreshes page data.
- */
-function updateResults() {
-  document.getElementById('search-list').innerHTML = '';
-  removeMarkers();
+  // Add event listeners to the cards for hovering.
+  const searchCards = document.getElementsByClassName('search-card');
+  for (let i = 0; i < searchCards.length; i++) {
+    const card = searchCards[i];
 
-  // Requery and repopulate page data.
-  const filter = document.getElementById('search-dropdown').value;
-  fetch('/discover/' + filter)
-    .then((data) => data.json())
-    .then((organizations) => {
-      createOrganizationCards(organizations);
-      createMarkers(organizations);
+    card.addEventListener('mouseover', function (e) {
+      selectCard(card.id);
+      card.dispatchEvent(new CustomEvent('cardHover', {bubbles: true, detail: card.id}));
     });
+
+    card.addEventListener('mouseout', function (e) {
+      selectCard();
+      card.dispatchEvent(new CustomEvent('cardHover', {bubbles: true, detail: null}));
+    });
+  }
 }
 
 /**
- * Renders the organization data into cards.
- * @param {JSON} organizations The JSON of organization data to add to the page.
+ * Selects and highlights an organization's card in the search list.
+ * @param {string} id The id of the organization whose card to select.
+ * @param {booolean} scroll Whether or not to scroll to the card.
  */
-function createOrganizationCards(organizations) {
-  const renderCards = Handlebars.compile(searchCardTemplate);
-  document.getElementById('search-list').innerHTML = renderCards({
-    organizations: organizations,
-  });
+export function selectCard(id = null, scroll = false) {
+  const cards = document.getElementsByClassName('search-card');
```

Comment:
Naming this same query as ```searchCards``` above.. either is fine, but let's keep it consistent?